core/src/main/java/com/linecorp/armeria/server/metric/MetricCollectingService.java

```diff
@@ -77,13 +78,17 @@ public static MetricCollectingServiceBuilder builder(MeterIdPrefixFunction meter
     private final MeterIdPrefixFunction meterIdPrefixFunction;
     @Nullable
     private final BiPredicate<? super RequestContext, ? super RequestLog> successFunction;
+    @Nullable
+    private final DistributionStatisticConfig distributionStatisticConfig;
```

Comment:
We passed the distStatCfg in here but where is it used? It looks to me that we don't actually pass it at all because the existing code simply gets it from `MoreMeters.distributionStatisticConfig()`?

Reply:
You're right. I've changed code without considering the design more seriously at this time. 🤔